build_runner/README.md

```diff
@@ -88,7 +89,7 @@ All the above commands support the following arguments:
 
 Some commands also have additional options:
 
-##### serve
+### serve
```

Comment:
```suggestion
##### serve
```

This is intended to describe the command line options for `serve`, so its a sub-heading of that

Reply:
The reasoning is, I changed them since it seemed hard to read in the readme as it blended in with the rest of the text.

Reply:
yeah the styling isn't ideal, but I think its important to keep the structure of the document correct

Reply:
Okay cool, I'll make these changes today.

Reply:
Actually I think I can just push a change to this PR so I will try that